test/integration/fixtures/blocks/core__media-text__image-alt-no-align.serialized.html

```diff
@@ -1,4 +1,4 @@
-<!-- wp:media-text {"align":"none","mediaType":"image"} -->
+<!-- wp:media-text {"mediaType":"image"} -->
```

Comment:
Now `none` is the default alignment, it will not be serialized into the block's comment delimiter.